### ISSUE TITLE
My question about the normalization of confusion matrix

### ISSUE THREAD

**Issue by dwuab** [AI-sandbox/XGMix#13]
In line 11 of `Utils/visualization.py`, the confusion matrix is calculated by `cm = confusion_matrix(y, y_pred)`. According to [sklearn's documentation](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html), the j-th element of i-th row is the number of observation whose true label is i but is predicted to be of label j. A natural way to normalize the confusion matrix would therefore be normalization by row sums.
However, in line 77 of `Utils/visualization.py`, the confusion matrix is normalized by the column sums: `cm = cm/np.sum(cm, axis=0)`. This does not seem right.
Am I wrong?
